### PR TITLE
Pyreverse: Don't show multiple class association arrows

### PR DESCRIPTION
**Pull request by nickdrozd**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Description

Fixes the multiple-association-arrows bug mentioned in #9045

### CODE REVIEW

File: doc/whatsnew/fragments/9045.bugfix [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Don't show multiple class association arrows.
```
Comment (Pierre-Sassoulas) [inline doc/whatsnew/fragments/9045.bugfix:1]:
```suggestion
Pyreverse doesn't show multiple class association arrows anymore, but only the strongest one.
```

**Review by Pierre-Sassoulas — COMMENTED** [review]
I feel like we need to determine what is the strongest association here and show only this one. There's another unrelated bug signaled by Andreas in the original issue (no composition here) that will make the association the only remaining link between those classes, right ?

**Comment by nickdrozd** [comment]
This change is a bit hacky. Rather than sort out why relations are being miscategorized, we just drop any extra arrows if they show up. This is done according to relation precedence, so if there is a composition arrow, we print it and then ignore any subsequent aggregation arrows, since those have lower precedence.

This doesn't address how relations are categorized, but it does eliminate the most annoying side effect of the problem, namely too many arrows.

**Review by DudeNr33 — APPROVED** [review]
I would be OK with separating the fixes for multiple arrows vs. determining the correct association into distinct PRs.

**Comment by Pierre-Sassoulas** [comment]
Wondering if we should release 3.0.0a8 without this or directly 3.0.0b0 with it (#9065 / #7607).

**Pull request merged**